src/main/java/sx/blah/discord/handle/audit/AuditLog.java

```diff
@@ -52,12 +52,14 @@ public AuditLog(LongMap<AuditLogEntry> entries) {
 	}
 
 	/**
-	 * Gets the entries for the log as a collection.
+	 * Gets the entries for the log as a list.
 	 *
-	 * @return The entries for the log as a collection.
+	 * @return The entries for the log as a list.
 	 */
-	public Collection<AuditLogEntry> getEntries() {
-		return entries.values();
+	public List<AuditLogEntry> getEntries() {
```

Comment:
This is a breaking API change that is not required. You may leave it as a `Collection` and that will not change the behavior of these changes.